scripts/install_python_prod_dependencies.py

```
@@ -458,7 +458,7 @@ def _get_possible_normalized_metadata_directory_names(
         )
         possible_names.add(
             normalize_directory_name(
-                '%s-%s-py3.10.egg-info' % (name, version_string)
+                '%s-%s-py3.14.egg-info' % (name, version_string)
```

Ruff (0.15.21)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)